cumulus/parachains/runtimes/collectives/collectives-westend/src/xcm_config.rs

```diff
@@ -0,0 +1,338 @@
+// Copyright (C) Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use super::{
+	AccountId, AllPalletsWithSystem, Balances, BaseDeliveryFee, FeeAssetId, Fellows, ParachainInfo,
+	ParachainSystem, PolkadotXcm, Runtime, RuntimeCall, RuntimeEvent, RuntimeOrigin,
+	TransactionByteFee, WeightToFee, XcmpQueue,
+};
+use frame_support::{
+	match_types, parameter_types,
+	traits::{ConstU32, Contains, Everything, Nothing},
+	weights::Weight,
+};
+use frame_system::EnsureRoot;
+use pallet_xcm::XcmPassthrough;
+use parachains_common::{impls::ToStakingPot, xcm_config::ConcreteAssetFromSystem};
+use polkadot_parachain_primitives::primitives::Sibling;
+use polkadot_runtime_common::xcm_sender::ExponentialPrice;
+use xcm::latest::prelude::*;
+use xcm_builder::{
+	AccountId32Aliases, AllowExplicitUnpaidExecutionFrom, AllowKnownQueryResponses,
+	AllowSubscriptionsFrom, AllowTopLevelPaidExecutionFrom, CurrencyAdapter,
+	DenyReserveTransferToRelayChain, DenyThenTry, EnsureXcmOrigin, FixedWeightBounds, IsConcrete,
+	LocatableAssetId, OriginToPluralityVoice, ParentAsSuperuser, ParentIsPreset,
+	RelayChainAsNative, SiblingParachainAsNative, SiblingParachainConvertsVia,
+	SignedAccountId32AsNative, SignedToAccountId32, SovereignSignedViaLocation, TakeWeightCredit,
+	TrailingSetTopicAsId, UsingComponents, WithComputedOrigin, WithUniqueTopic,
+};
+use xcm_executor::{traits::WithOriginFilter, XcmExecutor};
+
+const FELLOWSHIP_ADMIN_INDEX: u32 = 1;
+
+parameter_types! {
+	pub const WndLocation: MultiLocation = MultiLocation::parent();
+	pub const RelayNetwork: Option<NetworkId> = Some(NetworkId::Westend);
+	pub RelayChainOrigin: RuntimeOrigin = cumulus_pallet_xcm::Origin::Relay.into();
+	pub UniversalLocation: InteriorMultiLocation =
+		X2(GlobalConsensus(RelayNetwork::get().unwrap()), Parachain(ParachainInfo::parachain_id().into()));
+	pub CheckingAccount: AccountId = PolkadotXcm::check_account();
+	pub const GovernanceLocation: MultiLocation = MultiLocation::parent();
+	pub const FellowshipAdminBodyId: BodyId = BodyId::Index(FELLOWSHIP_ADMIN_INDEX);
+	pub AssetHub: MultiLocation = (Parent, Parachain(1000)).into();
+	pub AssetHubUsdtId: AssetId = (PalletInstance(50), GeneralIndex(1984)).into();
```

Comment:
not sure if `1984` is the Usdt id for westend asset hub

Reply:
Yes Joe created an asset with ID 1984